extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnWriterImpl.java

```diff
@@ -205,6 +217,7 @@ public void addVectorPage(
         }
         initWriter();
         // noinspection unchecked
+        // TODO Should I also change this path? I don't fully understand this path. Talk to Ryan for this.
```

Comment:
Almost certainly. This path (I think) is used for writing vectors or arrays, which effectively have one (nested) column of values, and one (outer) column of offsets into the values specifying the various vectors/arrays.